pyproject.toml

```
@@ -0,0 +1,82 @@
+[build-system]
+requires = ["setuptools", "pip >= 10"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pysatCDAAC"
+version = "0.0.4rc0"
```

```suggestion
version = "0.0.4"
```
oops. that shouldn't have been committed